themes/default/content/docs/intro/vs/terraform/terminology.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Terminology and Command Equivalence"
+meta_desc: "Pulumi is like Terraform: create, deploy, and manage infrastructure as code on any cloud. But unlike Terraform you can use familiar languages and tools."
+---
+
+<style>
+    main table {
+        font-size: 0.94em;
+        width: 100%;
+    }
+
+    main table th:first-child,
+    main table td:first-child {
+        width: 33%;
+    }
+</style>
+
+If you're already familiar with Terraform, learning Pulumi terminology and commands is simple. Many of the existing Terraform vocabulary and commands that you already know have direct equivalents in Pulumi. The table below lists common Terraform terms and CLI commands along with their Pulumi equivalents.
+
+## Terminology
+
+| Terraform | Pulumi |
+| --------- | ------ |
+| Workspace | [Stack]({{< relref "/docs/intro/concepts/stack" >}}) |
+| Variables | [Stack Config]({{< relref "/docs/intro/concepts/config" >}}) |
+| Directory | [Project]({{< relref "/docs/intro/concepts/project" >}}) |
+| Module | [Component]({{< relref "/docs/intro/concepts/resources/components" >}}) |
+| Resource | [Resource]({{< relref "/docs/intro/concepts/resources" >}}) |
+| Interpolation | [Interpolation]({{< relref "/docs/reference/cli/pulumi_up" >}}) |
```

Comment:
I think we want a different link for this one. Maybe this one? https://www.pulumi.com/docs/intro/concepts/inputs-outputs/#outputs-and-strings